connectors/google_drive/service_account.mdx

```diff
@@ -36,8 +36,8 @@ If you'd rather use an individuals account + OAuth to access Google Drive, check
   height="473"
   src="https://www.youtube.com/embed/Z5R9HUCVVAE?si=GZxteF6rmIHa1BOE"
   title="YouTube video player"
-  frameborder="0"
-  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+  frameBorder="0"
```

Comment:
Changed `frameborder` to `frameBorder` to resolve this error

![Uploading Screenshot 2024-01-12 at 12.17.45 AM.png…]()